Common subdirectories: a/MonoGame.Framework/Android and b/MonoGame.Framework/Android
Common subdirectories: a/MonoGame.Framework/Audio and b/MonoGame.Framework/Audio
Common subdirectories: a/MonoGame.Framework/Content and b/MonoGame.Framework/Content
Common subdirectories: a/MonoGame.Framework/Desktop and b/MonoGame.Framework/Desktop
diff -u a/MonoGame.Framework/GameComponentCollectionEventArgs.cs b/MonoGame.Framework/GameComponentCollectionEventArgs.cs
--- a/MonoGame.Framework/GameComponentCollectionEventArgs.cs	2012-03-27 10:37:06.000000000 -0700
+++ b/MonoGame.Framework/GameComponentCollectionEventArgs.cs	2012-04-29 18:20:18.606661757 -0700
@@ -40,7 +40,7 @@
 
 using System;
 
-﻿namespace Microsoft.Xna.Framework
+namespace Microsoft.Xna.Framework
 {
     public class GameComponentCollectionEventArgs : EventArgs
     {
Common subdirectories: a/MonoGame.Framework/GamerServices and b/MonoGame.Framework/GamerServices
Common subdirectories: a/MonoGame.Framework/Graphics and b/MonoGame.Framework/Graphics
Common subdirectories: a/MonoGame.Framework/Input and b/MonoGame.Framework/Input
Common subdirectories: a/MonoGame.Framework/Linux and b/MonoGame.Framework/Linux
Common subdirectories: a/MonoGame.Framework/MacOS and b/MonoGame.Framework/MacOS
Common subdirectories: a/MonoGame.Framework/Media and b/MonoGame.Framework/Media
Common subdirectories: a/MonoGame.Framework/Net and b/MonoGame.Framework/Net
Common subdirectories: a/MonoGame.Framework/Properties and b/MonoGame.Framework/Properties
diff -u a/MonoGame.Framework/ReusableItemList.cs b/MonoGame.Framework/ReusableItemList.cs
--- a/MonoGame.Framework/ReusableItemList.cs	2012-03-27 10:37:06.000000000 -0700
+++ b/MonoGame.Framework/ReusableItemList.cs	2012-04-29 18:20:37.683329204 -0700
@@ -25,7 +25,7 @@
 */
 #endregion License
 
-﻿using System;
+using System;
 using System.Collections.Generic;
 
 namespace Microsoft.Xna.Framework
Common subdirectories: a/MonoGame.Framework/Storage and b/MonoGame.Framework/Storage
Common subdirectories: a/MonoGame.Framework/Windows and b/MonoGame.Framework/Windows
Common subdirectories: a/MonoGame.Framework/iOS and b/MonoGame.Framework/iOS
